src/notifications/notificatielog/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
```

Comment:
Please remove empty/unused files

This is only picked up in two conditions:

1. You add `notificaties.notificatielog.apps.NotificatielogConfig` to `INSTALLED_APPS` instead of just `notificaties.notificatielog` (don't do that, it'll break the django-jenkins integration)
2. You define `default_app_config = 'notificaties.notificatielog.apps.NotificatielogConfig'` in `__init__.py`

None of these are done, so effectively you can actually remove this file.